packages/slate-react/src/plugin/react-editor.ts

```diff
@@ -572,36 +572,52 @@ export const ReactEditor = {
         // beginning of the next block
         if (
           IS_CHROME &&
-          anchorNode?.nodeType !== focusNode?.nodeType &&
+          anchorNode &&
+          focusNode &&
+          anchorNode.nodeType !== focusNode.nodeType &&
           domRange.anchorOffset === 0 &&
           domRange.focusOffset === 0 &&
-          focusNode?.nodeValue == null
+          focusNode.nodeValue == null
         ) {
           // If an anchorNode is an element node when triple clicked, then the focusNode
           //  should also be the same as anchorNode when triple clicked.
-          if (anchorNode!.nodeType === 1) {
-            focusNode = anchorNode
-          } else {
-            // Otherwise, anchorNode is a text node and we need to
-            // - climb up the DOM tree to get the farthest element node that receives
-            // triple click. It should have atribute 'data-slate-node' = "element"
-            // - get the last child of that element node
-            // - climb down the DOM tree to get the text node of the last child
-            // - this is also the end of the selection aka the focusNode
-            const anchorElement = anchorNode!.parentNode as HTMLElement
-            const tripleClickedBlock = anchorElement.closest(
-              '[data-slate-node="element"]'
-            )
-            const focusElement = tripleClickedBlock!.lastElementChild
-            // Get the element node that holds the focus text node
-            // Not every Slate element node contains a child node with `data-slate-string`,
-            // such as void nodes, so the result below could be null.
-            const innermostFocusElement = focusElement!.querySelector(
-              '[data-slate-string]'
-            )
-            if (innermostFocusElement) {
-              const lastTextNode = innermostFocusElement.childNodes[0]
-              focusNode = lastTextNode
+          // if (anchorNode!.nodeType === 1) {
```

Comment:
Just remove the commented out code.

Reply:
I'm not sure what you meant. Which part is the comment-out code?